Dice-Game/AI_Engine.cs

```diff
@@ -6,21 +6,24 @@
 
 namespace Dice_Game
 {
-    internal class AI_Engine
+    internal class Engine
     {
         private int highestDice { get; set; }
         private Random rnd;
 
-        public AI_Engine()
+        public Engine()
         {
-            highestDice = 19;
+            // Changed this up because Joe Bob always wants to win!
+            highestDice = -1;

```

Comment:
You should not change the number of the highest dice, simply because you want to win.